en/coding-the-legislation/writing_yaml_tests.md

```diff
@@ -156,23 +156,16 @@ Values can be arithmetic expressions too.
 
 ## Running a test
 
-The script `test_yaml.py` can run a single YAML test file, or all.
+To run yaml tests, use the command line tool `openfisca-run-test`, documented [here](http://openfisca.readthedocs.io/en/latest/openfisca-run-test.html) :
```

Comment:
YAML test is openfisca jargon, and I agree we can try to find something more meaninful.

However, the title of the page being "Writing YAML tests", I consider this to be out of the scope of this PR.